### PR TITLE
[Spacetime] Custom styles

### PR DESCRIPTION
**Pull request by jsanz**
**WIP**

Follow up from #428:

* Allows changing the language of the basemap
* Support for changing colors for the saturated basemap.

### CODE REVIEW

File: public/js/components/app.js [hunk 1]
```diff
@@ -223,13 +338,27 @@ export class App extends Component {
...
+                    colorOp={this.state.selectedColorOp}
+                    percentage={this.state.selectedPercentage}
+                  />
                 </EuiPageContentBody>
               </EuiPageContent>
               <EuiSpacer />
               <EuiPageContent>
                 <EuiPageContentBody>
-                  <LayerDetails title="Vector Layer" layerConfig={this.state.selectedFileLayer} />
+                  <LayerDetails
```
Comment (nickpeihl) [inline public/js/components/app.js:358]: When no Vector layers are selected we should either not display this component or add an empty state `message` to the `EuiInMemoryTable`. I think not displaying this component is easier and cleaner.

Comment (jsanz) [reply public/js/components/app.js]: Thanks for the feedback, addressed at [2662eac](https://github.com/elastic/ems-landing-page/pull/429/commits/2662eac7e6020db34eee0fbbcbadebd50b36211f).

**Comment by nickpeihl** [comment]
Due to time constraints, I think only basemap label translations will make it into Kibana 8.3. The basemap color filters may have to wait until 8.4. In that case, do you mind splitting the label translations and color operations from this PR into separate PRs?

**Comment by jsanz** [comment]
Closing this in favor of #434 for custom basemap languages and #435 for custom basemap colors.

**Pull request closed without merging**